### ISSUE TITLE
Image Comparisons Fail on PyQt6 6.0.1

### ISSUE THREAD

**Issue by j9ac9k** [pyqtgraph/pyqtgraph#1550]
PyQt6 6.0.1 was released a few hours ago, as our CI grabs the latest versions of PyQt6, it grabbed 6.0.1, and now we're having problems w/ our `assertImageApproved` checks.  I am able to replicate the failures locally.

Looking at the changelog for PyQt6, there is nothing here to suggest that there should be an issue here:

https://riverbankcomputing.com/news/PyQt_v6.0.1_Released

```
FAILED pyqtgraph/graphicsItems/tests/test_ImageItem.py::test_ImageItem - AssertionError
FAILED pyqtgraph/graphicsItems/tests/test_ImageItem.py::test_ImageItem_axisorder - AssertionError
FAILED pyqtgraph/graphicsItems/tests/test_ROI.py::test_getArrayRegion - AssertionError
FAILED pyqtgraph/graphicsItems/tests/test_NonUniformImage.py::test_NonUniformImage_lut - AssertionError
FAILED pyqtgraph/graphicsItems/tests/test_ROI.py::test_getArrayRegion_axisorder - AssertionError
FAILED pyqtgraph/graphicsItems/tests/test_NonUniformImage.py::test_NonUniformImage_colormap - AssertionError
FAILED pyqtgraph/graphicsItems/tests/test_ROI.py::test_PolyLineROI - AssertionError
```

EDIT: fair warning to anyone attempting to upgrade their python environment from PyQt6 6.0.0 to 6.0.1, doing `pip install --upgrade PyQt6` is insufficient, you would also need to install `PyQt6-Qt`

**Comment by pijyoi** [comment]
will be fixed by #1553

**Comment by ksunden** [comment]
Closing as 1553 is merged
